src/flb_output.c

```diff
@@ -856,6 +856,15 @@ int flb_output_init_all(struct flb_config *config)
                 flb_output_instance_destroy(ins);
                 return -1;
             }
+
+            /* Multi-threading enabled if configured */
+            ret = flb_output_enable_multi_threading(ins, config);
```

Comment:
Could you fix warning ?

```
/home/taka/git/tmp/3863/fluent-bit/src/flb_output.c: In function ‘flb_output_init_all’:
/home/taka/git/tmp/3863/fluent-bit/src/flb_output.c:861:19: warning: implicit declaration of function ‘flb_output_enable_multi_threading’ [-Wimplicit-function-declaration]
  861 |             ret = flb_output_enable_multi_threading(ins, config);
      |                   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
Thanks for pointing it out @nokute78 , have fixed the warning in recent commit